libdnf/hy-util.cpp

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2012-2013 Red Hat, Inc.
+ * Copyright (C) 2018 Bernhard Rosenkraenzer <bero@lindev.ch>
```

Comment:
Could you possibly remove this line? We prefer contributors to add themselves into the AUTHORS file rather than extending the file headers.

Reply:
done